src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/AceEditor.java

```diff
@@ -116,21 +116,21 @@ public boolean shouldComplete(NativeEvent event)
          Range range = getSession().getSelection().getRange();
```

Comment:
Let's try to avoid creating all these whitespace changes along with real changes (you can leave this one as-is though just a point for future reference)